### DISCUSSION TITLE
25.0.1

### DISCUSSION THREAD

**Discussion by davidjgraph** [jgraph/drawio-desktop#1921] [General]
## Releases Notes for 25.0.1
[Windows Installer](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/draw.io-25.0.1-windows-installer.exe)
[Windows No Installer](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/draw.io-25.0.1-windows-no-installer.exe)
[macOS - Universal](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/draw.io-universal-25.0.1.dmg)
Linux - [deb](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/drawio-amd64-25.0.1.deb), [AppImage](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/drawio-x86_64-25.0.1.AppImage) or [rpm](https://github.com/jgraph/drawio-desktop/releases/download/v25.0.1/drawio-x86_64-25.0.1.rpm)

Windows intel x32 releases are marked -ia32-

**ChangeLog:**

- Uses electron 32.2.5
- Updates to [draw.io core 25.0.1](https://github.com/jgraph/drawio/blob/v25.0.1/ChangeLog). All changes from 24.7.18 to 25.0.1 are added in this build.

<hr /><em>This discussion was created from the release <a href='https://github.com/jgraph/drawio-desktop/releases/tag/v25.0.1'>25.0.1</a>.</em>

**Comment by marybazo** [comment]
Hi!
I love your app!

However, I’m experiencing an issue with the latest version on a Mac with an M1 chip. The context menu doesn’t work when I try to open it. Instead, it behaves as if I’m trying to move the picture around the screen.

Here’s a screenshot of the menu I mean, taken from the previous version where it worked correctly.
<img width="532" alt="image" src="https://github.com/user-attachments/assets/f8ac59b3-8bd4-4683-95d4-74c2b4f7e4c5">

Have a nice day!

**Reply by alderg** [reply]
See https://github.com/jgraph/drawio-desktop/issues/1922
